SignalServiceKit/src/Messages/Attachments/TSAttachmentStream.m

```diff
@@ -330,7 +436,7 @@ + (void)deleteAttachments
 - (CGSize)calculateImageSize
 {
     if ([self isVideo]) {
-        return [self videoThumbnail].size;
+        return [self videoStillImage].size;
```

Comment:
I think we'll need to update `removeFileWithTransaction` to also remove the thumbnail.

Reply:
Good catch, done!